app/main.py

```diff
@@ -1,8 +1,21 @@
 class Person:
-    # write your code here
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[self.name] = self
     pass
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    person_list = [Person(element["name"], element["age"])
+                   for element in people]
+    for person in person_list:
+        for person_dict in people:
+            if person.name == person_dict["name"]:
+                if person_dict.get("wife") is not None:
+                    person.wife = Person.people[person_dict.get("wife")]
```

Comment:
If `person_dict.get('wife')` returns a name that does not exist in `Person.people`, this line will raise a `KeyError`. Consider using `Person.people.get(person_dict.get('wife'))` to safely attempt to retrieve the wife, which will return `None` if the name is not found.